packages/relay/src/lib/eth.ts

```diff
@@ -467,9 +467,9 @@ export class EthImpl implements Eth {
     const requestIdPrefix = formatRequestIdMessage(requestId);
     this.logger.trace(`${requestIdPrefix} estimateGas(transaction=${JSON.stringify(transaction)}, _blockParam=${_blockParam})`);
 
-    if ("data" in transaction){
-      this.counter.labels(EthImpl.ethEstimateGas, transaction.data.substring(0,10)).inc();
-    }
+    if (transaction?.data?.length >= 10)
```

Comment:
nit: for future can make 10 a const with a descriptive name e.g. functionSelectorCharLength

Reply:
✅  done